src/ray/gcs/format/gcs.fbs

```diff
@@ -194,6 +194,9 @@ table HeartbeatTableData {
   // Total resource capacity configured for this node manager.
   resources_total_label: [string];
   resources_total_capacity: [double];
+  // Aggregate outstanding resource load on this node manager.
+  resource_load_label: [string];
+  resource_load_capacity: [double];
```

Comment:
Nice, this will let us do queue based upscaling I suppose?

Reply:
yes, it should. It will capture aggregate resource load from all ready tasks queued on that raylet.

Reply:
you can even monitor load relative to the configured capacity (`resource_total_capacity`) and  set upscaling and downscaling watermarks in relative terms , e.g. load[cpu]/total[cpu] : 10x high watermark, 2x low watermark.